src/server/verification/verificationAPI.js

```diff
@@ -148,17 +149,17 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
       log.info('otp request:', { user, body })
 
       const mobile = body.user.mobile || user.otp.mobile
-
+      const hashedMobile = sha3(mobile)
       let userRec: UserRecord = _.defaults(body.user, user, { identifier: user.loggedInAs })
       const savedMobile = user.mobile
 
-      if (conf.allowDuplicateUserData === false && (await storage.isDupUserData({ mobile }))) {
+      if (conf.allowDuplicateUserData === false && (await storage.isDupUserData({ mobile: hashedMobile }))) {
```

Comment:
in storage it really didnt make much sense to allow an existing account to call /user/add, even in dev mode.
I just kept this flag in case we decide to go back to requiring uniqueness

Reply:
Understood,  thanks